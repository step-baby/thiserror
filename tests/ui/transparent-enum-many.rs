use thiserror::EnumDisplay;

#[derive(EnumDisplay, Debug)]
pub enum Error {
    #[display(transparent)]
    Other(anyhow::Error, String),
}

fn main() {}
