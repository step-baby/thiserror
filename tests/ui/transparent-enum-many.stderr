error: #[display(transparent)] requires exactly one field
 --> tests/ui/transparent-enum-many.rs:5:5
  |
5 | /     #[display(transparent)]
6 | |     Other(anyhow::Error, String),
  | |________________________________^
